lib/tape.rb

```diff
@@ -78,6 +78,14 @@ def tape_dir
     def local_dir
       Dir.pwd
     end
+
+    def tapefiles_dir
+      local_dir + '/taperole'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.